src/lib.rs

```diff
@@ -62,7 +62,7 @@ fn main() {
 
     // and since packet is ready, prepare the network part;
     // bind to some UDP socket
```

Comment:
Should be (something like that?):
`bind to UDP socket for supplied address and port in range`